src/main/scala/org/allenai/deep_qa/experiments/datasets/OmnibusDa.scala

```diff
@@ -0,0 +1,70 @@
+package org.allenai.deep_qa.experiments.datasets
```

Comment:
I'm tempted to have a `science` package under `datasets`, and eventually move all of the classes in `Science.scala` into files like this one.  The only real drawback I see to that is excessive nesting of packages.  What do you think?  If you think this is a good idea, you can make the new package, and move this file into it, and we can move other files over as we modify them.